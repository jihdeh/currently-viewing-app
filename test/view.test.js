import React from "react";
import App from "../src/app.jsx";
import renderer from "react-test-renderer";

test("updates displaying ip address", () => {
	const component =	renderer.create(
		<App />
	);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
