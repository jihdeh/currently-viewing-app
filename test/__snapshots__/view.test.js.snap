exports[`test updates displaying ip address 1`] = `
<div>
  <div
    className="container">
    <br />
    <div
      className="row">
      <div
        className="dual-list list-left col-md-5">
        <div
          className="well text-right">
          <div
            className="row">
            <div
              className="col-md-12">
              <h4>
                Your Details
              </h4>
            </div>
          </div>
          <ul
            className="list-group">
            <li
              className="list-group-item">
              Fetching your info .... 
            </li>
          </ul>
        </div>
      </div>
      <div
        className="dual-list list-right col-md-5">
        <div
          className="well">
          <div
            className="row">
            <div
              className="col-md-12">
              <h4>
                Visitor IP Adresses. 
                <small>
                  Including yours though.
                </small>
              </h4>
            </div>
          </div>
          <ul
            className="list-group">
            <li
              className="list-group-item">
              Loading ..... 
            </li>
          </ul>
          <p>
            Currently viewing: 
            0
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
